tests/pageObjects/AddCountryPage.js

```diff
@@ -0,0 +1,37 @@
+class AddCountryPage {
+    constructor(){
+        this.countryNameFieldSelector = "#country_name";
+        this.countryIsoNameFieldSelector = "#country_iso_name";
+        this.countryIsoFieldSelector = "#country_iso";
+        this.countryIso3FieldSelector = "#country_iso3";
+        this.stateRequiredSelector = "label[for='country_states_required']";
```

Comment:
use suitable name as possible
```suggestion
        this.countryStateSelector = "label[for='country_states_required']";
```